src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

```diff
@@ -302,6 +431,19 @@ public byte[] Read(int size)
             return ret;
         }
 
+        /// <summary>
+        /// Gets header of rpc port parcel.
+        /// </summary>
+        /// <returns>Parcel header</returns>
+        /// <since_tizen> 9 </since_tizen>
+        public ParcelHeader GetHeader()
+        {
+            Interop.LibRPCPort.Parcel.GetHeader(_handle, out IntPtr handle);
+            ParcelHeader header = new ParcelHeader();
+            header._handle = handle;
+            return header;
+        }
```

Comment:
가능하면 Property를 사용해주세요, 
cache를 적용해서 매번 객체를 안만들게 해주세요

```suggestion
        ParcelHeader _header;
        public ParcelHeader Header
        {
            get 
            {
                 if (_header != null)
                     return _header;
                     
                 Interop.LibRPCPort.Parcel.GetHeader(_handle, out IntPtr handle);
                 _header = new ParcelHeader() {
                       _handle = handle
                 };
                 return _header;
            }
        }
```